scipy/cluster/hierarchy.py

```diff
@@ -226,8 +226,17 @@ def _copy_arrays_if_base_present(T):
 
 
 def _randdm(pnts):
-    """ Generates a random distance matrix stored in condensed form. A
-        pnts * (pnts - 1) / 2 sized vector is returned.
+    """
+    Generate a random distance matrix stored in condensed form.
+
+    Parameters
+    ----------
+    pnts : int
+
+    Returns
+    -------
+    D : ndarray
+        A pnts * (pnts - 1) / 2 sized vector is returned.
```

Comment:
Can you put double backticks around ``pnts * (pnts - 1) / 2`` so it renders correctly?